internal/description/description.go

```diff
@@ -117,6 +117,7 @@ const (
 	ListOrganizationAPIKeys     = "List all organization API Keys for a given organization."
 	DescribeOrganizationsAPIKey = "Get a specific Organization API Key."
 	UpdateOrganizationsAPIKey   = "Update an Organization API Key."
+	CreateOrganizationsAPIKey   = "Create an Organization API Key for the specified organization."
```

Comment:
[nit] a bit redundant here
Either do `Create an API Key for the specified organization.`
Or `Create an Organization API Key.`